config/locales/zh.yml

```diff
@@ -1704,7 +1704,7 @@ two_factor_authentication.max_generic_login_attempts_reached: 为了你的安全
 two_factor_authentication.max_otp_login_attempts_reached: 为了你的安全，你的账户暂时被锁住，因为你错误输入一次性代码太多次。
 two_factor_authentication.max_otp_requests_reached: 为了你的安全，你的账户暂时被锁住，因为你要求一次性代码太多次。
 two_factor_authentication.max_personal_key_login_attempts_reached: 为了你的安全，你的账户暂时被锁住，因为你错误输入个人密钥太多次。
-two_factor_authentication.max_piv_cac_login_attempts_reached: 为了你的安全，你的账户暂时被锁住，因为你错误提供 piv/cac 凭据太多次。
+two_factor_authentication.max_piv_cac_login_attempts_reached: 为了你的安全，你的账户暂时被锁住，因为你错误提供 PIV/CAC 凭据太多次。
```

Comment:
![Screenshot 2024-11-07 at 4 16 43 PM](https://github.com/user-attachments/assets/82848bf6-be63-43fd-8576-d134dfb01dc0)

Arguably, this does not require LQA, because I am just correcting capitalization of this to match every other usage of it.